pkgs/by-name/oe/oelint-adv/Revert-urllib3-bump.patch

```diff
@@ -0,0 +1,26 @@
+From ab54d9fddd0cbc1a1088ce92ec5cc4927530724e Mon Sep 17 00:00:00 2001
+From: Otavio Salvador <otavio@ossystems.com.br>
+Date: Sat, 23 Sep 2023 21:48:22 -0300
+Subject: [PATCH] Revert urllib3 bump
+
+Signed-off-by: Otavio Salvador <otavio@ossystems.com.br>
+---
+ requirements.txt | 4 ++--
+ 1 file changed, 2 insertions(+), 2 deletions(-)
+
+diff --git a/requirements.txt b/requirements.txt
+index f64c62e..2817859 100644
+--- a/requirements.txt
++++ b/requirements.txt
+@@ -1,5 +1,5 @@
+ anytree ~= 2.9
+ colorama ~= 0.4
+ oelint-parser ~= 2.11
+-urllib3 ~= 2.0
+-wheel ~= 0.41
+\ No newline at end of file
++urllib3
++wheel ~= 0.41
```

Comment:
Can you please use `pythonRelaxDepsHook` instead?

Reply:
I tried but it fails as:

```
error: builder for '/nix/store/58wldb348bikm2f7kv3b6n1sfmyq75rb-oelint-adv-3.25.0.drv' failed with exit code 1;
       last 10 log lines:
       >   ir_d = dist.fetch_build_eggs(dist.install_requires)
       > WARNING: The directory '/homeless-shelter/.cache/pip' or its parent directory is not owned or is not writable by the current user. The cache has been disabled. Check the permissions and owner of that directory. If executing pip with sudo, you should use sudo's -H flag.
       > WARNING: Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<pip._vendor.urllib3.connection.HTTPSConnection object at 0x7ffff580e620>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution')': /simple/urllib3/
       > WARNING: Retrying (Retry(total=3, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<pip._vendor.urllib3.connection.HTTPSConnection object at 0x7ffff580e410>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution')': /simple/urllib3/
       > WARNING: Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<pip._vendor.urllib3.connection.HTTPSConnection object at 0x7ffff580f460>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution')': /simple/urllib3/
       > WARNING: Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<pip._vendor.urllib3.connection.HTTPSConnection object at 0x7ffff580f610>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution')': /simple/urllib3/
       > WARNING: Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<pip._vendor.urllib3.connection.HTTPSConnection object at 0x7ffff580f7c0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution')': /simple/urllib3/
       > ERROR: Could not find a version that satisfies the requirement urllib3~=2.0 (from versions: none)
       > ERROR: No matching distribution found for urllib3~=2.0
       > error: Command '['/nix/store/lx8vhp4fxclp494svlfis3sb2g8z4l9h-python3-3.10.12/bin/python3.10', '-m', 'pip', '--disable-pip-version-check', 'wheel', '--no-deps', '-w', '/build/tmp04vys396', '--quiet', 'urllib3~=2.0']' returned non-zero exit status 1.
```

Reply:
Fixed.